server/.env

```diff
@@ -4,3 +4,5 @@ FLASK_APP=project.__init__.py
 DATABASE_URL=postgres://postgres:password@postgres-db:5432/db_dev
 DATABASE_TEST_URL=postgres://postgres:password@postgres-db:5432/db_test
 DATABASE_SECRET=myprecious
+OAUTH_CLIENT_ID=798726661513-p69g4p5hn1cmura8mgm0sth9kqlnjoke.apps.googleusercontent.com
+OAUTH_CLIENT_SECRET=w_1s_mqk1bZ1G0iFnmMNefCG
```

Comment:
Should we be committing these remotely? Yes it is private but this is bad practice.

Reply:
We could gitignore the env file

Reply:
we should not!